pkg/controlplane/controller/kubernetesservice/controller.go

```diff
@@ -38,7 +38,7 @@ import (
 )
 
 const (
-	kubernetesServiceName = "kubernetes"
+	KubernetesServiceName = "kubernetes"
```

Comment:
we don't need these changes in this controller

Reply:
Undone.